src/mlpack/tests/main_tests/hmm_train_test.cpp

```diff
@@ -64,7 +64,7 @@ inline void CheckMatricesDiffer(arma::mat& a, arma::mat& b, double tolerance)
   bool valsEqual = true;
   if (dimsEqual)
   {
-    for (size_t i=0; i<a.n_elem; i++)
+    for (size_t i=0; i<a.n_elem; ++i)
```

Comment:
```suggestion
    for (size_t i = 0; i < a.n_elem; ++i)
```
Just a style fix I saw while looking through. :)